src/Translation/locales/es.json

```diff
@@ -103,6 +103,15 @@
     "your_estate": "Su propiedad",
     "your_parcel": "Tu parcela"
   },
+  "auction_modal": {
+    "authorize_contract": "Contrato de autorización",
+    "authorized_explanation": "Estás listo para comenzar! ",
```

Comment:
Oops., It is different from English translation and also there is an extra space at the end `"Estás listo para comenzar! "'